apps/web/src/utils/asset-validation.ts

```diff
@@ -72,6 +72,12 @@ export const validateClaimAsset = (t: Translate) =>
     packTemplateId: packTemplateId(t),
   })
 
+export const validateRevokeAsset = (t: Translate) =>
+  object({
+    packId: packId(t),
+    ownerId: string(required(t('forms:errors.required') as string)),
+  })
```

Comment:
This currently requires an `ownerId`, but the revoke pack schema includes a `fromAddress` that could be used optionally if we wanted to transfer from an address which wasn't connected to a user in the database. Though this is the front-end validation, which does require the owner ID for now.